templates/partials/actionbar.html

```diff
@@ -97,13 +97,19 @@
               <span class="fa fa-envelope"></span>
               <p translate>Send Message</p>
             </a>
-            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="!actionBar.right.verified" ng-click="emit('VerifyReport', true)">
+            <div class="dropdown-menu sub-actions" mm-auth="can_verify_reports" ng-show="actionBar.right.verifyingReport">
+              <a class="mm-icon mm-icon-caption verify-error" ng-class="{ active: actionBar.right.verified && !actionBar.right.verified_valid }" ng-click="emit('VerifyReport', false)">
+                <span class="reports-icon reports-icon-verify-error"></span>
```

Comment:
"reports-icon-verify-error" is a very explicit class. Can we just use "verify-error" and then use `.reports-icon.verify-error` in the css to achieve the same result but with a class name that's easier to reuse in other places?